### ISSUE TITLE
Add "Remove from Merge" to context menu in Merge Details pane

### ISSUE THREAD

**Issue by erlunsford** [matortheeternal/merge-plugins#73]
This would make it much easier to remove an esp from a merge instead of having to hunt it down in the Plugins tab

**Comment by matortheeternal** [comment]
So you're suggesting the Details Popup Menu (which currently only has the "Copy to clipboard" menu item) would have a new item that would conditionally show only when a user right clicks a plugin in a merge to remove that plugin from the merge.

That's a reasonable idea.

**Comment by erlunsford** [comment]
yes, exactly. Wasn't sure the effort involved. Thanks for accepting the idea.
